### ISSUE TITLE
PDF build of inbo_rapport fails

### ISSUE THREAD

**Issue by florisvdh** [inbo/INBOmd#58]
Tried to compile [inbo/inbo_mdexamples@5ed3192](https://github.com/inbo/inbomd_examples/tree/5ed31921041a3871ee22553efc0a32a998fd4065/source/inbo_rapport) (current master) > directory `source/inbo_rapport` (from within RStudio -> Build: `bookdown::pdf_book`).

This happened:

```
output file: inbo_rapport.knit.md

/usr/lib/rstudio/bin/pandoc/pandoc +RTS -K512m -RTS inbo_rapport.utf8.md --to latex --from markdown+autolink_bare_uris+tex_math_single_backslash --output inbo_rapport.tex --template /home/floris/lib/R/library/INBOmd/pandoc/inbo_rapport.tex --variable documentclass=report --variable codesize=footnotesize --variable style=inbo_report --variable mylanguage=french,english,dutch --pdf-engine xelatex --wrap preserve --natbib --variable toc=TRUE --variable number_sections=TRUE --variable fig_caption=TRUE --variable tables=yes --standalone 
! Undefined control sequence.
\ULset ->\UL@setULdepth \def \UL@leadtype 
                                          {\leaders \hrule \@height \UL@heig...
l.39 \maketitle

Error: LaTeX failed to compile inbo_rapport.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See inbo_rapport.log for more info.
Execution halted

Exited with status 1.
```
[These notes](https://tex.stackexchange.com/questions/100378/uline-within-a-pgfmath-expression-produces-an-error/100384) may be relevant.

<details> <summary><strong>Session Info</strong></summary>

```r
─ Session info ─────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.6.3 (2020-02-29)
 os       Linux Mint 18.1             
 system   x86_64, linux-gnu           
 ui       RStudio                     
 language (EN)                        
 collate  nl_BE.UTF-8                 
 ctype    nl_BE.UTF-8                 
 tz       Europe/Brussels             
 date     2020-03-23                  

─ Packages ─────────────────────────────────────────────────────────────────
 package     * version date       lib source                          
 assertthat    0.2.1   2019-03-21 [1] CRAN (R 3.6.0)                  
 bookdown      0.18    2020-03-05 [1] CRAN (R 3.6.3)                  
 cli           2.0.2   2020-02-28 [1] CRAN (R 3.6.3)                  
 crayon        1.3.4   2017-09-16 [1] CRAN (R 3.6.0)                  
 digest        0.6.25  2020-02-23 [1] CRAN (R 3.6.3)                  
 dplyr         0.8.5   2020-03-07 [1] CRAN (R 3.6.3)                  
 drat          0.1.5   2019-03-28 [1] CRAN (R 3.6.0)                  
 evaluate      0.14    2019-05-28 [1] CRAN (R 3.6.1)                  
 fansi         0.4.1   2020-01-08 [1] CRAN (R 3.6.2)                  
 glue          1.3.2   2020-03-12 [1] CRAN (R 3.6.3)                  
 htmltools     0.4.0   2019-10-04 [1] CRAN (R 3.6.1)                  
 INBOmd        0.4.7   2020-02-10 [1] Github (inbo/INBOmd@886140d)    
 knitr         1.28    2020-02-06 [1] CRAN (R 3.6.3)                  
 magrittr      1.5     2014-11-22 [1] CRAN (R 3.6.0)                  
 packrat       0.5.0   2018-11-14 [1] CRAN (R 3.6.0)                  
 pillar        1.4.3   2019-12-20 [1] CRAN (R 3.6.2)                  
 pkgconfig     2.0.3   2019-09-22 [1] CRAN (R 3.6.1)                  
 purrr         0.3.3   2019-10-18 [1] CRAN (R 3.6.1)                  
 qrcode        0.1.2   2019-08-01 [1] Github (thierryo/qrcode@378d76b)
 R.methodsS3   1.8.0   2020-02-14 [1] CRAN (R 3.6.3)                  
 R.oo          1.23.0  2019-11-03 [1] CRAN (R 3.6.1)                  
 R.utils       2.9.2   2019-12-08 [1] CRAN (R 3.6.2)                  
 R6            2.4.1   2019-11-12 [1] CRAN (R 3.6.2)                  
 Rcpp          1.0.4   2020-03-17 [1] CRAN (R 3.6.3)                  
 rlang         0.4.5   2020-03-01 [1] CRAN (R 3.6.3)                  
 rmarkdown     2.1     2020-01-20 [1] CRAN (R 3.6.3)                  
 rstudioapi    0.11    2020-02-07 [1] CRAN (R 3.6.3)                  
 sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.6.0)                  
 tibble        2.1.3   2019-06-06 [1] CRAN (R 3.6.0)                  
 tidyselect    1.0.0   2020-01-27 [1] CRAN (R 3.6.2)                  
 withr         2.1.2   2018-03-15 [1] CRAN (R 3.6.0)                  
 xfun          0.12    2020-01-13 [1] CRAN (R 3.6.2)                  
 yaml          2.2.1   2020-02-01 [1] CRAN (R 3.6.2)                  

[1] /home/floris/lib/R/library
[2] /usr/local/lib/R/site-library
[3] /usr/lib/R/site-library
[4] /usr/lib/R/library
```
</details>

**Comment by ThierryO** [comment]
This is fixed in the hotfix branch https://github.com/inbo/INBOmd/tree/hotfix

**Comment by ThierryO** [comment]
Fixed in #59
